core/app.h

```diff
@@ -289,13 +289,19 @@ namespace MR
             opt (option), args (arguments)
         {
           for (size_t i = 0; i != option->size(); ++i) {
```

Comment:
@Lestropie: this bit warrants closer inspection to make sure it still works as originally intended...

Reply:
Presumably you mean the whole code block below? You can click & drag on the "new comment" + button to select a set of lines & associate a comment thread with it.

Will do some tests. The message could potentially be expanded a bit also.

Reply:
Basic test performs as expected; this is including 505adef8:

```
mrcat sub-109639_T2w.nii.gz sub-109639_desc-preproc_T2w.nii.gz -datatype -axis 3 test.mif
mrcat: [WARNING] Value "-axis" is being used as the expected argument for option "-datatype", yet this itself looks like a separate command-line option; the requisite input to command-line option "-datatype" may have been erroneously omitted, which may cause other command-line parsing errors
mrcat: [ERROR] unknown format for image "3"
mrcat: [ERROR] error opening image "3"
```